src/auth/model/git-hub-user-dto.spec.ts

```diff
@@ -13,7 +13,7 @@ describe('AccessToken', () => {
     const ghUser = new GitHubUserDto({
       login: 'foo',
       name: 'myname',
-      avatar_url: 'http://my.avatar.example.com',
+      'avatar_url': 'http://my.avatar.example.com',
```

Comment:
Why the change?

Reply:
Seems eslint is a little persnickety on some of these issues -- these shouldn't need to be quoted. Not that it matters, but seems a bit over the top!